pybedtools/test/data/issue319.out.bed

```diff
@@ -0,0 +1,3496 @@
+##contig=<ID=chr19_KI270931v1_alt,length=170148>
+##contig=<ID=chr19_KI270932v1_alt,length=215732>
+##contig=<ID=chr19_KI270933v1_alt,length=170537>
+##contig=<ID=chr19_GL000209v2_alt,length=177381>
+##contig=<ID=chr3_KI270934v1_alt,length=163458>
+##contig=<ID=chr6_GL000253v2_alt,length=4677643>
+##contig=<ID=chr19_GL949749v2_alt,length=1091841>
+##contig=<ID=chr3_KI270935v1_alt,length=197351>
+##contig=<ID=chr6_GL000254v2_alt,length=4827813>
+##contig=<ID=chr19_GL949750v2_alt,length=1066390>
+##contig=<ID=chr3_KI270936v1_alt,length=164170>
+##contig=<ID=chr6_GL000255v2_alt,length=4606388>
+##contig=<ID=chr19_GL949751v2_alt,length=1002683>
+##contig=<ID=chr3_KI270937v1_alt,length=165607>
+##contig=<ID=chr6_GL000256v2_alt,length=4929269>
+##contig=<ID=chr19_GL949752v1_alt,length=987100>
+##contig=<ID=chr6_KI270758v1_alt,length=76752>
+##contig=<ID=chr19_GL949753v2_alt,length=796479>
+##contig=<ID=chr19_KI270938v1_alt,length=1066800>
+##contig=<ID=chrEBV,length=171823>
+##contig=<ID=chrUn_KN707606v1_decoy,length=2200>
+##contig=<ID=chrUn_KN707607v1_decoy,length=3033>
+##contig=<ID=chrUn_KN707608v1_decoy,length=3112>
+##contig=<ID=chrUn_KN707609v1_decoy,length=1642>
+##contig=<ID=chrUn_KN707610v1_decoy,length=1393>
+##contig=<ID=chrUn_KN707611v1_decoy,length=1103>
+##contig=<ID=chrUn_KN707612v1_decoy,length=1039>
+##contig=<ID=chrUn_KN707613v1_decoy,length=1619>
+##contig=<ID=chrUn_KN707614v1_decoy,length=3122>
+##contig=<ID=chrUn_KN707615v1_decoy,length=1934>
+##contig=<ID=chrUn_KN707616v1_decoy,length=3111>
+##contig=<ID=chrUn_KN707617v1_decoy,length=2545>
+##contig=<ID=chrUn_KN707618v1_decoy,length=2295>
+##contig=<ID=chrUn_KN707619v1_decoy,length=1551>
+##contig=<ID=chrUn_KN707620v1_decoy,length=2046>
+##contig=<ID=chrUn_KN707621v1_decoy,length=1222>
+##contig=<ID=chrUn_KN707622v1_decoy,length=1535>
+##contig=<ID=chrUn_KN707623v1_decoy,length=3784>
+##contig=<ID=chrUn_KN707624v1_decoy,length=1329>
+##contig=<ID=chrUn_KN707625v1_decoy,length=1238>
+##contig=<ID=chrUn_KN707626v1_decoy,length=5623>
+##contig=<ID=chrUn_KN707627v1_decoy,length=5821>
+##contig=<ID=chrUn_KN707628v1_decoy,length=2960>
+##contig=<ID=chrUn_KN707629v1_decoy,length=1848>
+##contig=<ID=chrUn_KN707630v1_decoy,length=2315>
+##contig=<ID=chrUn_KN707631v1_decoy,length=1945>
+##contig=<ID=chrUn_KN707632v1_decoy,length=1424>
+##contig=<ID=chrUn_KN707633v1_decoy,length=1274>
+##contig=<ID=chrUn_KN707634v1_decoy,length=1007>
+##contig=<ID=chrUn_KN707635v1_decoy,length=1414>
+##contig=<ID=chrUn_KN707636v1_decoy,length=1725>
+##contig=<ID=chrUn_KN707637v1_decoy,length=5354>
+##contig=<ID=chrUn_KN707638v1_decoy,length=2189>
+##contig=<ID=chrUn_KN707639v1_decoy,length=1294>
+##contig=<ID=chrUn_KN707640v1_decoy,length=1831>
+##contig=<ID=chrUn_KN707641v1_decoy,length=1647>
+##contig=<ID=chrUn_KN707642v1_decoy,length=2943>
+##contig=<ID=chrUn_KN707643v1_decoy,length=2857>
+##contig=<ID=chrUn_KN707644v1_decoy,length=1030>
+##contig=<ID=chrUn_KN707645v1_decoy,
```

Comment:
In the rendered text here,`Chdiak-Higashi_syndrome` does not appear to be unicode (it's missing the `é`). Is this due to github rendering? Or is the file missing the the problematic unicode (in which case a passing test is misleading?)

Reply:
Hi Ryan @daler !

Thanks for you suggestions. Sorry for the delay. Fixing this is still a priority for us in bcbio.

I've simplified the error handling and removed unnecessary lines from the test vcf header.
The test works - without a fix it breaks pybedtools, with the fix it passes.

The problematic symbol is rendered as 
![problematic_symbol](https://user-images.githubusercontent.com/13981803/80402169-81ef5980-888b-11ea-84bd-331686f86f54.png)
in the terminal or in Atom.

So it is in the file - it breaks pybedtools when given as input.

Please merge.
Sergey